mlflow/store/artifact/databricks_artifact_repo.py

```diff
@@ -211,10 +211,10 @@ def _azure_upload_file(self, credentials, local_file, artifact_path):
                             "Failed to authorize request, possibly due to credential expiration."
                             " Refreshing credentials and trying again..."
                         )
-                        credential_info = self._get_write_credential_infos(
+                        credentials = self._get_write_credential_infos(
```

Comment:
It's to reuse it on line 222 instead of getting it again. 
If the credential expires, there's no need to get it in both put_block nad put_block_list steps.